ds_judgements_public_ui/sass/includes/_govuk_overrides.scss

```
@@ -186,3 +186,90 @@
 .govuk-form-group {
   margin-bottom: 0;
 }
+
+.govuk-header {
+  border-bottom: none;
+}
+
+.govuk-header__link--homepage:hover,
+.govuk-header__link--homepage:active {
+  border-bottom: $space-1 solid $color-black;
+}
+
+.govuk-header__navigation-item--active a,
+.govuk-header__navigation-item--active a:hover,
+.govuk-header__navigation-item--active a:visited {
+  color: $color-yellow;
+  font-weight: bold !important;
```

If we make this styling more specific, we can remove the `!important`:

```suggestion
.govuk-header__navigation-item--active .govuk-header__link,
.govuk-header__navigation-item--active .govuk-header__link:hover,
.govuk-header__navigation-item--active .govuk-header__link:visited {
  color: $color-yellow;
  font-weight: bold;
```